6608 - Docs changes for backup target mount options. (#800)
longhorn/website: content/docs/1.6.0/snapshots-and-backups/backup-and-restore/set-backup-target.md

If you don't get any error messages, try creating a backup and confirm the content is pushed out to your new bucket.

### Set up a Local Testing Backupstore

> Longhorn provides sample backupstore server setups for testing purposes.  They are in `longhorn/deploy/backupstores`.  There are examples for S3 (MinIO), Azure, NFS, and CIFS.

Reviewer: ```suggestion
Longhorn provides sample backupstore server setups for testing purposes. You can find samples for AWS S3 (MinIO), Azure, CIFS, and NFS in the `longhorn/deploy/backupstores` folder.  
```

> 1. Use the following command to set up a MinIO S3 server for the backupstore after `longhorn-system` is created.

Reviewer: ```suggestion
1. Set up a MinIO S3 server for the backupstore in the `longhorn-system` namespace.  
```

### Set up NFS Backupstore

> If using NFS as a backupstore, the NFS server must support NFSv4.

Reviewer: ```suggestion
Ensure that the NFS server supports NFSv4 and that the target URL points to the service.  
```

> The target URL should point to the service like this:

Reviewer: ```suggestion
Example:  
```

```
nfs://longhorn-test-nfs-svc.default:/opt/backupstore
```

> The default mounting options are `actimeo=1,soft,timeo=300,retry=2`.  It is possible to override those options with your own by appending the keyword "nfsOptions" and the options string to the target URL.  For example:  

Reviewer: ```suggestion
The default mount options are `actimeo=1,soft,timeo=300,retry=2`. To use other options, append the keyword "nfsOptions" and the options string to the target URL.  

Example:  
```

```
nfs://longhorn-test-nfs-svc.default:/opt/backupstore?nfsOptions=soft,timeo=330,retrans=3  
```

> Note that if any mount options are supplied, they will replace, not augment, the built-in options.

Reviewer: ```suggestion
Any mount options that you specify will replace, not add to, the default options.  
```

    ```
    cifs://longhorn-test-cifs-svc.default/backupstore
    ```

> 	As with NFS, it is possible to override the client mount options by extending the URL with "cifsOptions."  For example,

Reviewer: ```suggestion
The default CIFS mount option is "soft". To use other options, append the keyword "cifsOptions" and the options string to the target URL.  

Example:  
```

    ```
    cifs://longhorn-test-cifs-svc.default/backupstore?cifsOptions=rsize=65536,wsize=65536,soft
    ```

> 	The default CIFS mount option is "soft".

Reviewer: ```suggestion
Any mount options that you specify will replace, not add to, the default options.  
```
